src/content/protocols/pwn.md

```diff
@@ -0,0 +1,140 @@
+| PWNSimpleLoanSimpleProposal | makeProposal | Makes an onchain proposal. Checks in this function are derived from _makeProposal. | Proposer |
+| PWNSimpleLoanSimpleProposal | acceptProposal | Accepts a given proposal. Checks in this function are derived from _acceptProposal. | Loan contract specified in an offer |
+| PWNSimpleLoanListProposal | _makeProposal | Marks a proposalHash as valid. Caller has to also be a proposer of the corresponding offer. | Proposer |
+| PWNSimpleLoanListProposal | _acceptProposal | Accepts a given proposal. Loan contracts call this function with the signed proposal on behalf of the acceptor. Malicious actor could accept proposals for different Loan contracts. | Loan contract specified in an offer |
+| PWNSimpleLoanListProposal | makeProposal | Makes an onchain proposal. Checks in this function are derived from _makeProposal. | Proposer |
+| PWNSimpleLoanListProposal | acceptProposal | Accepts a given proposal. Checks in this function are derived from _acceptProposal. | Loan contract specified in an offer |
+| PWNSimpleLoanElasticProposal | _makeProposal | Marks a proposalHash as valid. Caller has to also be a proposer of the corresponding offer. | Proposer |
+| PWNSimpleLoanElasticProposal | _acceptProposal | Accepts a given proposal. Loan contracts call this function with the signed proposal on behalf of the acceptor. Malicious actor could accept proposals for different Loan contracts. | Loan contract specified in an offer |
+| PWNSimpleLoanElasticProposal | makeProposal | Makes an onchain proposal. Checks in this function are derived from _makeProposal. | Proposer |
+| PWNSimpleLoanElasticProposal | acceptProposal | Accepts a given proposal. Checks in this function are derived from _acceptProposal. | Loan contract specified in an offer |
+| PWNSimpleLoanDutchAuctionProposal | _makeProposal | Marks a proposalHash as valid. Caller has to also be a proposer of the corresponding offer. | Proposer |
+| PWNSimpleLoanDutchAuctionProposal | _acceptProposal | Accepts a given proposal. Loan contracts call this function with the signed proposal on behalf of the acceptor. Malicious actor could accept proposals for different Loan contracts. | Loan contract specified in an offer |
+| PWNSimpleLoanDutchAuctionProposal | makeProposal | Makes an onchain proposal. Checks in this function are derived from _makeProposal. | Proposer |
+| PWNSimpleLoanDutchAuctionProposal | acceptProposal | Accepts a given proposal. Checks in this function are derived from _acceptProposal. | Loan contract specified in an offer |
+| PWNUtilizedCredit | utilizeCredit | Updates utilised credit for an account. This is used to share credit between different proposal types. Malicious actor could update the utilised credit to go over the credit limit. | Contract with a valid tag in PWNHub |
+
+
+## Dependencies
+
+No external dependency has been found.
+
```

Comment:
The docs mention Chainlink Oracles for the elastic chainlink proposals. Could you quickly explain here how chainlink is used.